wiki/Welcome/fr.md

```diff
@@ -0,0 +1,41 @@
+[//]: # (Traduit par cth103)
```

Comment:
remove the translator info.

it's **not needed**. since github's blame/history would tell us about the author.